ensime_shared/ensime.py

```diff
@@ -465,16 +369,16 @@ def send_at_point_req(self, what, path, row, col, size, where="range"):
              where: {"from": i, "to": i + size}})
 
     def do_toggle_teardown(self, args, range=None):
-        self.log("do_toggle_teardown: in")
+        self.log.debug('do_toggle_teardown: in')
         self.toggle_teardown = not self.toggle_teardown
 
     def type_check_cmd(self, args, range=None):
         """Sets the flag to begin buffering typecheck notes & clears any
         stale notes before requesting a typecheck from the server"""
-        self.log("type_check_cmd: in")
+        self.log.debug('type_check_cmd: in')
         self.start_typechecking()
         self.type_check("")
-        self.vim.command('silent !echo "Typechecking..."')
+        self.editor.message('typechecking')
```

Comment:
Oh, I also changed this in the course of the logging PR #323 that should go through before this anyway.